plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -195,10 +196,15 @@ func (r *NodeAuthorizer) authorizeReadNamespacedObject(nodeName string, starting
 		klog.V(2).Infof("NODE DENY: '%s' %#v", nodeName, attrs)
 		return authorizer.DecisionNoOpinion, "cannot read subresource", nil
 	}
-	if len(attrs.GetNamespace()) == 0 {
+	if len(attrs.GetNamespace()) == 0 && namespaced {
 		klog.V(2).Infof("NODE DENY: '%s' %#v", nodeName, attrs)
 		return authorizer.DecisionNoOpinion, "can only read namespaced object of this type", nil
 	}
+	if len(attrs.GetNamespace()) != 0 && !namespaced {
+		klog.V(2).Infof("NODE DENY: '%s' %#v", nodeName, attrs)
+		return authorizer.DecisionNoOpinion, "can only read cluster object of this type", nil
+	}
```

Comment:
this bit doesn't matter, a cluster scoped object *can't* get read within a namespace, drop this block